adhocracy-plus/assets/scss/components/_live_questions.scss

```diff
@@ -85,3 +85,7 @@
     position: relative;
     bottom: -7*$spacer;
 }
+
+.livequestion__detail--image {
```

Comment:
Sorry, I didn't see that yesterday, but the image is not a variant. A variant needs to be used together with the thing it's a variant from like `btn` and `btn-primary`. Öhm, I need to look BEM up again, should it be `livequestion__detail__image`?

Reply:
Ok, I decided that livequestion-detail is the block as livequestion__detail wasn't yet used. And image is the element, so it's `livequestion-detail__image` now.

Reply:
yes, makes sense, the **B**lock is `livequestion-detail` and the **E**lement is `image`. thanks for changing it :) and sorry that i paid less attention to the naming here.